controllers/grants.go

```diff
@@ -81,6 +84,21 @@ func ClaimGrant(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	conn := datastore.GetRedisConn(r.Context())
+
+	if err == nil {
+		// FIXME TODO clean this up via a better abstraction
+		if _, err := redis.Int((*conn).Do("ZINCRBY", "count:claimed:ip", "1", r.RemoteAddr)); err != nil {
+			raven.CaptureMessage("Could not increment claim count for ip.", map[string]string{"IP": r.RemoteAddr})
```

Comment:
Not sure how the db is designed, but may be it'll be good to capture the error string with the IP?

Reply:
`CaptureMessage` takes a string message and an optional dictionary of additional metadata. Sentry treats each distinct string as a different event type, but does keep track of the additional metadata.